plugins/wordcompletion/Makefile.am

```diff
@@ -0,0 +1,53 @@
+# Word Completion plugin
+plugindir = $(PLUMA_PLUGINS_LIBS_DIR)
+
+AM_CPPFLAGS = \
+	-I$(top_srcdir) 				\
+	$(PLUMA_CFLAGS) 				\
+	$(WARN_CFLAGS)
```

Comment:
here was another compilation flag before my last force push:
`$(DISABLE_DEPRECATED_CFLAGS)`
thanks for your help :)

Reply:
Np :)
how does the plugin work?
It's installed and enabled but i don't see how to use it?
I tried with HTML syntax but nothing happens.